mbf_abstract_nav/include/mbf_abstract_nav/abstract_controller_execution.h

```diff
@@ -358,6 +364,9 @@ namespace mbf_abstract_nav
     //! current robot pose;
     geometry_msgs::PoseStamped robot_pose_;
 
+    //! current robot actual velocity;
+    geometry_msgs::TwistStamped robot_velocity;
```

Comment:
Please respect convention: `robot_velocity_`

Reply:
I will change it.